README.md

```diff
@@ -29,26 +29,30 @@ cf restage $YOUR_APP_NAME
 
 #### Log Collection (Beta - No official release yet)
 
-To start collecting logs from your application in CloudFoundry, the agent contained in the buildpack needs to be activated and log collection enabled.
+To start collecting logs from your application in CloudFoundry, use the following configuration:
 
 ```
+# activate the agent
 cf set-env $YOUR_APP_NAME RUN_AGENT true
+# enable log collection
 cf set-env $YOUR_APP_NAME DD_LOGS_ENABLED true
-# Disable the Agent core checks to disable system metrics collection
+# add a custom config
+cf set-env $YOUR_APP_NAME LOGS_CONFIG '[{"type":"tcp","port":"<PORT>","source":"<SOURCE>","service":"<SERVICE>"}]'
```

Comment:
Is this always necessary? Can we make it clearer if not

Reply:
If it's not is this a breaking change? We will need to be very clear about this in the docs if so

Reply:
It's a breaking change and we're planning to be very clear, we were clear that it was a private beta and that the integration was likely to change